Plugins/SwiftLintPlugin/SwiftLintPlugin.swift

```diff
@@ -186,13 +183,124 @@ struct SwiftLintPlugin: BuildToolPlugin {
     }
     // swiftlint:enable function_body_length
 
+    // MARK: - Standalone tool Main
+#if !canImport(PackagePlugin)
+
+    private func getModifiedFiles(at path: Path) -> [Path] {
+        let task = Process("/usr/bin/git", ["diff", "HEAD", "--name-only"])
```

Comment:
This occurrence is probably not critical, but I think we should be getting tools paths using `which`. I'm using git from `/opt/homebrew/bin/git`.

I have a handy [Process extension](https://github.com/duckduckgo/macos-symbolication/blob/main/Sources/symbolicate/Extensions/Process%2BWhich.swift) that you could borrow :)